### PR TITLE
Fixes #13: Spring Framework Kata update to Spring 6.0.8 and Java 17. 

### LINKED ISSUE #13: Update Spring Framework Kata to use Spring 6.x.x and Java 17

### PR DESCRIPTION
**Pull request by cguntur-bnym**
Associated changes to dependencies and imports.

### CODE REVIEW

File: spring-framework/pom.xml [hunk 1]
```diff
@@ -34,18 +34,25 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- The properties below ensure that the code is picked up and compiled as Java 11 -->
```
Comment (donraab) [inline spring-framework/pom.xml:36]: Should this comment be updated to Java 17?

File: spring-framework/ch08_spring-rest/pom.xml [hunk 2]
```diff
@@ -43,18 +42,7 @@
         <maven.compiler.source>11</maven.compiler.source>
```
Comment (donraab) [inline spring-framework/ch08_spring-rest/pom.xml:42]: Should source and target now be 17 instead of 11 here?

**Review by donraab — CHANGES_REQUESTED** [review]
Hi @cguntur-bnym, the changes look good to me, but I have a couple of minor questions inline. Thanks!

**Review by prathasirisha — APPROVED** [review]

**Comment by cguntur-bnym** [comment]
Can someone merge this, please?

**Pull request merged**